apps/openmw/mwmechanics/aiescort.cpp

```diff
@@ -23,10 +23,11 @@
 
 namespace MWMechanics
 {
-    AiEscort::AiEscort(const std::string &actorId, int duration, float x, float y, float z)
+    AiEscort::AiEscort(const std::string &actorId, int duration, float x, float y, float z, bool repeat)
     : mActorId(actorId), mX(x), mY(y), mZ(z), mRemainingDuration(static_cast<float>(duration))
     , mCellX(std::numeric_limits<int>::max())
     , mCellY(std::numeric_limits<int>::max())
+    , mRepeat(repeat)

```

Comment:
Thank you for your comment. That was the way I was originally doing it, but in order for the calls to the various AI constructors from aiextensions.cpp (for running an AI command from the console or script) to set the variable to false, it seemed to me this way the only way to do it. Am I overlooking something?